src/common/webview_chromium.cpp

```diff
@@ -1793,6 +1863,18 @@ CefRefPtr<CefResourceHandler> ClientHandler::GetResourceHandler(
     return m_resourceManager->GetResourceHandler(browser, frame, request);
 }
 
+void
+ClientHandler::ShowDevTools(CefRefPtr<CefBrowser> browser,
```

Comment:
Is there a reason why basic `ShowDevTools()` is not public? I think it may be handy to be able to show the tool programmatically.

The `wxWebViewEdge` and `wxWebViewWebKit` also support this, see https://github.com/PBfordev/wxecharts/blob/c3f50dc464308b623272f345b06071a1212f2f4a/mainframe.cpp#L308. I can't use macOS so I don't know about that one.

Reply:
Looks like a good idea, thanks, I think I can add it. For now let's keep it simple and just allow opening it without specifying the position of the element to inspect, we can always extend it later if really necessary.